pkgs/os-specific/linux/lxc/default.nix

```diff
@@ -58,6 +58,7 @@ stdenv.mkDerivation rec {
     "localstatedir=\${TMPDIR}"
     "sysconfdir=\${out}/etc"
     "sysconfigdir=\${out}/etc/default"
+    "bashcompdir=\${out}/share/bash-completion/completions/lxc"
```

Comment:
The directory is `$out/share/bash-completion/completions`, *not* `$out/share/bash-completion/completions/lxc`-